runtime/src/gpu/cuda/gpu-cuda.c

```diff
@@ -73,6 +75,14 @@ static void chpl_gpu_ensure_context() {
 void chpl_gpu_impl_init() {
   int         num_devices;
 
+  CHPL_GPU_DEBUG("Initing GPU layer. (Memtype: %s)\n",
+#ifdef CHPL_GPU_MEM_UVA
+      "uva"
```

Comment:
I want to avoid using the term 'uva' for when we mean page-locked host memory (that can be accessed by the GPU via DMA).  I think I'd prefer we print out messages like this:

```
Memory allocation strategy for ---
    array data: unified memory
         other: unified memory
```

Or this:

```
Memory allocation strategy for ---
    array data: device memory
         other: page-locked host memory
```